### PR TITLE
EFS-1290 added more links to Person and Patient UI pages

### PR DESCRIPTION
**Pull request by drcate**

### CODE REVIEW

File: src/pages/resources/Patient.tsx [hunk 1]
```diff
@@ -263,12 +263,24 @@ const Patient = ({ resource }: { resource: TPatient }): React.ReactElement => {
                 resourceType={resource.resourceType}
                 reverseReferences={[{'target': 'CareTeam', 'property': 'patient'}]}
             />
+            <Partials.ReverseReference
+                name="Composition"
+                id={uuid}
+                resourceType={resource.resourceType}
+                reverseReferences={[{'target': 'Composition', 'property': 'person'}]}
```
Comment (007sunny) [inline src/pages/resources/Patient.tsx:270]:
```suggestion
                reverseReferences={[{'target': 'Composition', 'property': 'patient'}]}
```

Comment (drcate) [reply src/pages/resources/Patient.tsx:270]: All of the links in the subject field in Composition are of the form 
Patient/person.9af5bbbf-76d5-52d2-89a0-9509e30802b5
So person is needed here.
patient does not work.

Comment (007sunny) [reply src/pages/resources/Patient.tsx:270]: The link made by this is '[base]/4_0_0/Composition?person=Patient/[id]'

And as per FHIR spec 'person' is not a valid search parameter for Composition (https://www.hl7.org/fhir/r4b/composition.html#search) so the filter is being ignored in the query (`db.Composition_4_0_0.find({'meta.tag':{'$not':{'$elemMatch':{'system':'https://fhir.icanbwell.com/4_0_0/CodeSystem/server-behavior','code':'hidden'}}}}, {}).sort({'_uuid':1}).limit(10)`) and all Compositions are returned.

Comment (drcate) [reply src/pages/resources/Patient.tsx]: Removed Composition link from Patient page

**Review by 007sunny — APPROVED** [review]

**Pull request merged**